### ISSUE TITLE
Can't install hhmv on debian jessie

### ISSUE THREAD

**Issue by SonicGD** [facebook/hhvm#4616]
Prebuilded hhvm package depends on libjpeg8, but there no such package in jessie anymore, cause debian is migrating to libjpeg-turbo.

As you can see - https://packages.debian.org/jessie/libjpeg8

```
apt-get install hhvm-nightly
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 hhvm-nightly : Depends: libjpeg8 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

On existing installations i still have libjpeg8, but on fresh servers it's a problem.

**Comment by spacek** [comment]
Exactly the same problem for me.

**Comment by willdurand** [comment]
:+1:

**Comment by paulbiss** [comment]
Please open an issue on https://github.com/hhvm/packaging

**Comment by anthonyfok** [comment]
For the record, the removal of libjpeg8 from Debian was *not* due to migration to libjpeg-turbo.
The original libjpeg series and libjpeg-turbo lives along together just fine in Debian to this day.

Rather, the direct reason is that libjpeg8 was superseded by libjpeg9, and the Debian package maintainer decided to request libjpeg8's removal: https://bugs.debian.org/867047

The latest and final Debian libjpeg8 package may be found in the Debian snapshot archive at this link: https://snapshot.debian.org/package/libjpeg8/8d1-2/

Meanwhile, Ubuntu has kept libjpeg8 available to this day in 20.04 LTS and in groovy for maximum backward compatiblity.
